scripts/coder-dev.sh

```diff
@@ -10,6 +10,7 @@ source "${SCRIPT_DIR}/lib.sh"
 
 GOOS="$(go env GOOS)"
 GOARCH="$(go env GOARCH)"
+debug="${debug:-0}"
```

Comment:
Wondering if this name is too common for a shell var that changes behavior? Maybe `DEBUG_DELVE` or something similar like that?

Reply:
Done!